lib/suspenders/app_builder.rb

```diff
@@ -205,6 +214,11 @@ def configure_background_jobs_for_rspec
       run 'rails g delayed_job:active_record'
     end
 
+    def configure_background_jobs_for_rspec_mongoid
+      copy_file 'background_jobs_rspec.rb', 'spec/support/background_jobs.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.